Quantum-Control-Applications/Superconducting/Single Flux Tunable Transmon/configuration.py

```diff
@@ -46,45 +46,95 @@ def IQ_imbalance(g, phi):
 pi_len = 100
 pi_amp = 0.05
 
+drag_coef = 0
+anharmonicity = -200 * u.MHz
+AC_stark_detuning = 0 * u.MHz
+
 gauss_len = 200
 gauss_sigma = gauss_len / 5
 gauss_amp = 0.25
 gauss_wf = gauss_amp * gaussian(gauss_len, gauss_sigma)
 
-x180_len = 100
+x180_len = 40
 x180_sigma = x180_len / 5
-x180_amp = 0.15
-x180_wf, x180_der_wf = np.array(drag_gaussian_pulse_waveforms(x180_amp, x180_len, x180_sigma, alpha=0, delta=1))
+x180_amp = 0.35
+x180_wf, x180_der_wf = np.array(
+    drag_gaussian_pulse_waveforms(
+        x180_amp, x180_len, x180_sigma, alpha=drag_coef, anharmonicity=anharmonicity, detuning=AC_stark_detuning
+    )
+)
+x180_I_wf = x180_wf
+x180_Q_wf = x180_der_wf
 # No DRAG when alpha=0, it's just a gaussian.
 
 x90_len = x180_len
 x90_sigma = x90_len / 5
 x90_amp = x180_amp / 2
-x90_wf, x90_der_wf = np.array(drag_gaussian_pulse_waveforms(x90_amp, x90_len, x90_sigma, alpha=0, delta=1))
+x90_wf, x90_der_wf = np.array(
+    drag_gaussian_pulse_waveforms(
+        x90_amp, x90_len, x90_sigma, alpha=drag_coef, anharmonicity=anharmonicity, detuning=AC_stark_detuning
+    )
+)
+x90_I_wf = x90_wf
+x90_Q_wf = x90_der_wf
 # No DRAG when alpha=0, it's just a gaussian.
 
-x270_len = x180_len
-x270_sigma = x270_len / 5
-x270_amp = -x90_amp
-x270_wf, x270_der_wf = np.array(drag_gaussian_pulse_waveforms(x270_amp, x270_len, x270_sigma, alpha=0, delta=1))
+minus_x90_len = x180_len
+minus_x90_sigma = minus_x90_len / 5
+minus_x90_amp = -x90_amp
+minus_x90_wf, minus_x90_der_wf = np.array(
+    drag_gaussian_pulse_waveforms(
+        minus_x90_amp,
+        minus_x90_len,
+        minus_x90_sigma,
+        alpha=drag_coef,
+        anharmonicity=anharmonicity,
+        detuning=AC_stark_detuning,
+    )
+)
+minus_x90_I_wf = minus_x90_wf
+minus_x90_Q_wf = minus_x90_der_wf
 # No DRAG when alpha=0, it's just a gaussian.
 
 y180_len = x180_len
 y180_sigma = y180_len / 5
 y180_amp = x180_amp
-y180_wf, y180_der_wf = np.array(drag_gaussian_pulse_waveforms(y180_amp, y180_len, y180_sigma, alpha=0, delta=1))
+y180_wf, y180_der_wf = np.array(
+    drag_gaussian_pulse_waveforms(
+        y180_amp, y180_len, y180_sigma, alpha=drag_coef, anharmonicity=anharmonicity, detuning=AC_stark_detuning
```

Comment:
I think it's better to remove the `alpha=`, `anharmonicity=` and `detuning=`.

Reply:
I guess that this is subjective, because I personally prefer to have the parameters names written explicitly otherwise I don't know which one is anharmonicity or detuning... I know that it is written in the docstring, but most of the time people don't look at it especially when running experiments if you want to change parameters quickly.
Maybe @KevinAVR can pick a side...

Reply:
I'm ok either way, don't really care. 
Your arguments are good, I'll give mine, but then just decide whatever way you prefer :)
1. I think that if the names of the variables are meaningful (as they are here), then it's ok. It does say: `alpha=drag_coef` and `anharmonicity=anharmonicity` so the information there is doubled.
2. Not having it makes it backward compatible with older versions of `qualang_tools`. So if some customer goes now to qualibs and downloads the files (without updating `qualang_tools`), it would work for him. (Currently it won't).

Reply:
Ok you convinced me, in this case since the variable names are explicit we don't need to keep the parameter's names :)